.github/workflows/CD.yml

```diff
@@ -629,16 +629,34 @@ jobs:
           src: public.ecr.aws/${{ env.ECR_REPO }}:${{ needs.release-checking.outputs.version }}
           dst: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ needs.release-checking.outputs.version }}
 
+  mirror_adot_operator:
+    runs-on: ubuntu-latest
+    needs: release-latest-image
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17.7
+
+      - name: Compare version with Dockerhub latest
+        id: version
+        if: needs.release-latest-image.outputs.cache-hit != 'true'
+        run: |
+          TAG="${{ needs.release-checking.outputs.version }}"
+          TARGET_VERSION=$TAG bash tools/workflow/docker-version-compare.sh
+
       - name: Configure AWS Credentials
-        if: steps.release-latest-image.outputs.cache-hit != 'true'
+        if: needs.release-latest-image.outputs.cache-hit != 'true'
```

Comment:
It does not look like you export this from the `release-latest-image` job.

Reply:
Added cache step and changed conditional to check against that instead